backend/src/services/ai_analysis.py

```
@@ -126,24 +139,20 @@ def _parse_llm_analysis(self, analysis_content: str) -> Dict:
         lines = analysis_content.split('\n')
         for line in lines:
             if "Sentiment:" in line:
-                parsed_data["sentiment"] = line.split("Sentiment:")[1].strip().replace("[","").replace("]","")
+                parsed_data["sentiment"] = line.split("Sentiment:")[1].strip().replace("[","").replace("]","").replace("\"","").replace(",","")
             elif "Probability Score:" in line:
                 try:
-                    score_str = line.split("Probability Score:")[1].strip().replace("[","").replace("]","")
+                    score_str = line.split("Probability Score:")[1].strip().replace("[","").replace("]","").replace("\"","").replace(",","")
                     parsed_data["probability_score"] = int(score_str.split(" ")[0])
-                except ValueError: pass
+                except (ValueError, IndexError): pass
```

Ruff (0.15.2)

[error] Multiple statements on one line (colon)

(E701)